_positions/director-of-outreach.md

```diff
@@ -29,3 +29,5 @@ You’re a broad and strategic thinker who can see the interconnectivity between
 - Define team and individual performance metrics and expected business results through OKRs and regular calibration of team goals with individual goals.
 
 
+##[Apply to join 18F](https://jobs.lever.co/18f/1e419ac8-737d-4edb-b5ab-f186775246f1/apply)
```

Comment:
Sure, that sounds good. (I kind of liked the additional context that the singular vs. plural adds, as it tells the applicant whether there will be other people with the same role or whether they'll be the only one, but the position/role description itself should adequately cover that information.)